tooling/eslint/nextjs.js

```diff
@@ -3,7 +3,6 @@ const config = {
   extends: ["plugin:@next/next/recommended"],
   rules: {
     "@next/next/no-html-link-for-pages": "off",
-    "@typescript-eslint/require-await": "off",
```

Comment:
This rule is deactivated as server actions requires to be async but they might not have await statements